spec/system/account_notes_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Account notes', :inline_jobs, :js, :streaming do
+  include ProfileStories
+
+  let(:email)               { 'test@example.com' }
+  let(:password)            { 'password' }
+  let(:confirmed_at)        { Time.zone.now }
+  let(:finished_onboarding) { true }
+
+  let!(:other_account) { Fabricate(:account) }
+
+  before { as_a_logged_in_user }
+
+  it 'can be written and viewed' do
+    visit_profile(other_account)
+
+    note_text = 'This is a personal note'
+    fill_in 'Click to add note', with: note_text
+
+    # This is a bit awkward since there is no button to save the change
+    # The easiest way is to send ctrl+enter ourselves
+    find_field(class: 'account__header__account-note__content').send_keys [:control, :enter]
+
+    expect(page)
+      .to have_css('.account__header__account-note .inline-alert', text: 'SAVED')
+
+    expect(page)
+      .to have_css('.account__header__account-note__content', text: note_text)
+
+    # This is extra awkward because the “Saved” message is actually shown before the API request
+    # returns. Maybe we should change that?
+    sleep 0.1
+
+    expect(AccountNote.find_by(account: bob.account, target_account: other_account).comment)
+      .to eq note_text
```

Comment:
How about moving this expectation to the bottom of the example then? Would this work without the `sleep`?

Reply:
Let's try, but I'm afraid this would just cause one of the other tests to be flaky, since the request could be aborted by the `visit root_url`